willdurand/geocoder-bundle/5.0/config/packages/bazinga_geocoder.yaml

```diff
@@ -0,0 +1,17 @@
+bazinga_geocoder:
+    # The local IP (127.0.0.1) will be replaced by the fake_ip
+    # see https://github.com/geocoder-php/BazingaGeocoderBundle/blob/5.0.0/Resources/doc/index.md#fake-local-ip
+    fake_ip: 123.123.123.123
+
+    # The following config requires 'composer req geocoder-php/chain-provider geocoder-php/google-maps-provider geocoder-php/geo-plugin-provider'
```

Comment:
I'm not sure if this commented block is useful. Without reading the docs, I'm not I would be able to understand what's going on. Instead, I would add a comment with a link to the docs as the very first line of this file.

Reply:
Thank you.